salt/api-gateway/init.sls

```diff
@@ -151,22 +152,30 @@ kong-db-exists:
 #
 #
 
-kong-init-script:
+kong-upstart-script:
     file.managed:
         - name: /etc/init/kong.conf
         - source: salt://api-gateway/config/etc-init-kong.conf
         - template: jinja
 
+kong-systemd-script:
+    file.managed:
+        - name: /lib/systemd/system/kong.service
```

Comment:
by the way are we sure that `/lib/systemd/system` is available as a folder in 14.04 or should the state have a `makedirs` attribute